.buildscript/androidVersions.gradle

```diff
@@ -0,0 +1,8 @@
+ext {
+    compileVersion = 25
+    toolsVersion = "25.0.3"
+    minVersion = 14
+    targetVersion = 25
+    libVersionCode = 1
```

Comment:
The way I see only the compileSdkVersion, buildToolsVersion and targetSdkVersion should be extracted as the other properties never change.